### PR TITLE
Introscreen -Cooper

### PR DESCRIPTION
**Pull request by chanson02**
Added my intro screen.
@NoahBartell @MLHumphries please branch off from this (or I guess you could also just commit to this branch too!) to implement your intro screens.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -96,26 +97,34 @@ def userSelectedCharacter(self, character: Character):
...
         if character.name == "John":
+            self.TypeWriterSound = pygame.mixer.Sound(AUDIO["TypeWriterSound"])
+            self.TypeWriterSound.set_volume(0.9)
+            self.TypeWriterSound.play(loops=-1)
             # Load John's theme music
             pygame.mixer.music.load(AUDIO["JohnTheme"])
             # Set the volume to maximum (1.0)
             pygame.mixer.music.set_volume(0.4)
             # Play John's theme music in a loop indefinitely
             pygame.mixer.music.play(-1)
```
Comment (chanson02) [inline main.py]: This is johns portion where his introscreen will show.

File: main.py [hunk 2]
```diff
@@ -96,26 +97,34 @@ def userSelectedCharacter(self, character: Character):
...
 
-        self.characterWidget = CharacterDetailWidget(
-            character, transition_callback=self.showMapScreen
-        )
+            self.characterWidget = CharacterDetailWidget(
+                character, transition_callback=self.showMapScreen
+            )
+        elif character.name == "Cooper":
+            self.characterWidget = CooperIntroScreen(character, transition_callback=self.showMapScreen)
```
Comment (chanson02) [inline main.py]: Here is where mine will show.

File: main.py [hunk 3]
```diff
@@ -96,26 +97,34 @@ def userSelectedCharacter(self, character: Character):
...
+        else:
+            raise Exception(f'Introscreen for {character.name} has not yet been implemented')
```
Comment (chanson02) [inline main.py]: @NoahBartell @MLHumphries make sure to make your own and set it to `self.characterWidget`

File: dunwoody_disaster/saves/Mitch.json
Comment (chanson02) [inline dunwoody_disaster/saves/Mitch.json:1]: note to self: delete this file.

Comment (MLHumphries) [reply dunwoody_disaster/saves/Mitch.json:1]: Hey hey hey now, I was only helping to catch a small grammatical error. That is my contribution to this project.

Comment (chanson02) [reply dunwoody_disaster/saves/Mitch.json:1]: I applaud your efforts.

**Review by EngineeringDude123 — APPROVED** [review]
Lookin good looking good

**Comment by chanson02** [comment]
This is ready to be merged after #72.

**Review by instMeyFTW — APPROVED** [review]

**Review by NoahBartell — APPROVED** [review]
yuh

**Review by MLHumphries — APPROVED** [review]

**Pull request merged**